ci/taos/plugins-staging/pr-nnstreamer-ubuntu-apptest.sh

```diff
@@ -0,0 +1,143 @@
+    echo -e "[MODULE] TAOS/pr-nnstreamer-apptest: Starting sample app test"
+    check_dependency cmake
+    check_dependency make
+    check_dependency wget
+    check_dependency python
+
+    # Set-up environment variables.
+    export NNST_ROOT="${dir_ci}/${dir_commit}/${PRJ_REPO_OWNER}"
+    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$NNST_ROOT/lib
+    export GST_PLUGIN_PATH=$GST_PLUGIN_PATH:$NNST_ROOT/lib
+    declare -i result=0
+
+    # Build and install nnstreamer library
+    cd ${NNST_ROOT}
+    if [[ -d ./build ]]; then
+        rm -rf ./build/*
+    else
+        mkdir build
+    fi
+    cd build
+    cmake -DCMAKE_INSTALL_PREFIX=${NNST_ROOT} \
+    -DINCLUDE_INSTALL_DIR=${NNST_ROOT}/include \
+    -DGST_INSTALL_DIR=${NNST_ROOT}/lib ..
+    make install
+    cd ..
+
+    # Set-up testing environment.
+    mkdir bin
+    cp build/nnstreamer_example/example_filter/nnstreamer_example_filter bin/
+    cp nnstreamer_example/example_filter/nnstreamer_example_filter.py bin/
+    cp build/nnstreamer_example/example_cam/nnstreamer_example_cam bin/
+    cp build/nnstreamer_example/example_sink/nnstreamer_sink_example bin/
+    cp build/nnstreamer_example/example_sink/nnstreamer_sink_example_play bin/
+    rm -rf build
+    cd bin    
+
+    # Download tensorflow-lite model file and labels.
+    mkdir tflite_model
+    cd tflite_model
+    wget https://github.com/nnsuite/testcases/tree/master/DeepLearningModels/tensorflow-lite/Mobilenet_1.0_224_quant/mobilenet_v1_1.0_224_quant.tflite
+    result+=$?
+    wget https://raw.githubusercontent.com/nnsuite/testcases/master/DeepLearningModels/tensorflow-lite/Mobilenet_1.0_224_quant/labels.txt
+    result+=$?
+    cd ..
+
+    if [[ ${result} -ne 0 ]]; then
+        echo -e "[DEBUG][FAILED] Oooops!!!!!! apptest is failed."
+        echo -e " The data files was not downloaded. Please check the log file to get a hint"
+        echo -e ""
+        check_result="failure"
+        global_check_result="failure"
+
+        message="Oooops. apptest is failed. Resubmit the PR after fixing correctly. Commit number is $input_commit."
+        cibot_pr_report $TOKEN "failure" "TAOS/pr-nnstreamer-apptest" "$message" "$REPOSITORY_WEB/pull/$input_pr/commits/$input_commit" "$GITHUB_WEBHOOK_API/statuses/$input_commit"
+    
+        # comment a hint on failed PR to author.
+        message=":octocat: **cibot**: $user_id, apptest could not be completed. To find out the reasons, please go to ${CISERVER}/${PRJ_REPO_UPSTREAM}/ci/${dir_commit}/checker-pr-audit.log"
+        cibot_comment $TOKEN "$message" "$GITHUB_WEBHOOK_API/issues/$input_pr/comments"
+
+        return ${result}
+    fi
+    
+    # Test with sample apps
+    # @todo We have to modify the commented test files below because this script requires a USB camera device.
+    # https://github.com/nnsuite/nnstreamer/issues/531
+    # - [RunTest] fake USB camera for NNStreamer video apps
+    
+    # ./nnstreamer_example_filter
```

Comment:
Let's display debug message as following:
1. incase of using USB Camera (e.g., fake usb cameare or replacing v4l2src with videosrc), ....
   - ./nnstreamer_example_filter , python nnstreamer_example_filter.py
   - ./nnstreamer_example_cam
2. in case of  not using USB Camera, ....
   - ./nnstreamer_sink_example
   - ./nnstreamer_sink_example_play (Need to check if this file requires USB Camera.)

Reply:
# Let's check if /dev/video0 device noe is enabled.
if [[ -f /dev/video0 ]] ; then
    echo -e "[DEBUG] USB Camera device is enabled. It is required by {nnstreamer_example_filter|nnstreamer_example_cam}."
else 
    echo -e "[DEBUG] USB Camera deivce is not enabled. It is required by {nnstreamer_example_filter|nnstreamer_example_cam}."